applications/zpc/components/zwave_command_classes/assets/ZWave_custom_cmd_classes.xml

```diff
@@ -12998,10 +13001,10 @@
       <param key="0x00" name="Color Component ID" type="BYTE" />
       <param key="0x01" name="Current Value" type="BYTE" />
       <param key="0x02" name="Target Value" type="BYTE" />
-      <param key="0x03" name="Duration" type="CONST">
-        <const key="0x00" flagname="Already at the Target Value" flagmask="0x00" />
-        <const key="0x01" flagname="Unknown duration" flagmask="0xFE" />
-        <const key="0x02" flagname="Reserved" flagmask="0xFF" />
+      <param key="0x03" name="Duration" type="BYTE">
```

Comment:
May this be in separate change, ideally we should avoid duplication of this file, and rely on a public version from zwa , this is something to be discussed .

Reply:
Related topic:

https://github.com/Z-Wave-Alliance/zwave_xml/issues/88

https://github.com/Z-Wave-Alliance/zwave_xml/issues/103

Let me mark this resolved since it is merged